freqtrade/rpc/webhook.py

```diff
@@ -0,0 +1,66 @@
+"""
+This module manages webhook communication
+"""
+import logging
+from typing import Any,  Dict
+
+from requests import post, RequestException
+
+from freqtrade.rpc import RPC, RPCMessageType
+
+
+logger = logging.getLogger(__name__)
+
+logger.debug('Included module rpc.webhook ...')
+
+
+class Webhook(RPC):
+    """  This class handles all webhook communication """
+
+    def __init__(self, freqtrade) -> None:
+        """
+        Init the Webhook class, and init the super class RPC
+        :param freqtrade: Instance of a freqtrade bot
+        :return: None
+        """
+        super().__init__(freqtrade)
+
+        self._config = freqtrade.config
+
+    def cleanup(self) -> None:
+        """
+        Cleanup pending module resources.
+        This will do nothing for webhooks, they will simply not be called anymore
+        """
+        pass
+
+    def send_msg(self, msg: Dict[str, Any]) -> None:
+        """ Send a message to telegram channel """
+        try:
+
+            if msg['type'] == RPCMessageType.BUY_NOTIFICATION:
+                valuedict = self._config['webhook'].get('webhookbuy', None)
```

Comment:
might be a bit more readable (imho) to do  in `__init__` e.g.
```
self._webhook_buy = self._config['webhook'].get('webhookbuy', None)
```
and use the private variables instead